try {
    var name_1 = myFunction();
    console.log("Name:", name_1);
}
catch (error) {
    console.error("Error occured:", error.message);
}
